src/dataviews/dataview-model-base.js

```diff
@@ -185,6 +185,7 @@ module.exports = Model.extend({
   // Class props
   {
     ATTRS_NAMES: [
+      'type',
```

Comment:
The type should be a static attr and set by a concrete implementation (e.g. see the [category dataview](https://github.com/CartoDB/cartodb.js/blob/3b6dcfd9e68e952142b387d03d3bc61ba4e2b13c/src/dataviews/category-dataview-model.js#L18)), it's not supposed to be changeable (which it would be by adding the attr name here).

Using the type+cid was a temporary hack, now that widgets are created (in editor) when persisted we should probably pass through the id (i.e. add it here instead of type), so server logs actually can be traceable back to the widget definition sources. cc @rochoa 

If the ids are not important for Windshaft, or if widgets are created ad-hoc (sales?), the proper place would be to change the [initialize](https://github.com/CartoDB/cartodb.js/blob/3b6dcfd9e68e952142b387d03d3bc61ba4e2b13c/src/dataviews/dataview-model-base.js#L37), to get the type from `this.defaults.type`.